labs/src/main/scala/org/scalalabs/basic/lab03/OptionExercise.scala

```diff
@@ -37,6 +42,15 @@ object OptionExercise02 {
    * to convert a possible numeric String (e.g. Some("12")) to an integer
    */
   def totalPeopleInRooms(rooms: Map[Int, Option[String]]): Int = {
-    error("Fix me")
+      val capacities = rooms.values.map( capacity _ )
+      capacities.sum
+  }
+  
+  def capacity(room: Option[String]) : Int = {
+      val number = room.getOrElse("0")
+      if (number == "locked")
+          0
+      else
+          number.toInt
```

Comment:
I like your verboseness with _val_ declarations. While unnecessary, this really helps readability. That being said, there are some places I feel your code would be more Scala-like if you omitted some () and . operators.